signed/verifiers.go

```diff
@@ -157,6 +163,9 @@ type RSAPyCryptoVerifier struct{}
 // with PyCrypto.
 func (v RSAPyCryptoVerifier) Verify(key data.PublicKey, sig []byte, msg []byte) error {
 	digest := sha256.Sum256(msg)
+	if key.Algorithm() != data.RSAKey {
+		return ErrInvalidKeyType{}
+	}
 
 	k, _ := pem.Decode([]byte(key.Public()))
```

Comment:
Enforcing a minimum key length for RSA on gotuf seems perfectly appropriate. At least you should enforce that the size is > 1024. There are no legitimate uses of < 1024, and this would prefer people from slipping by an invalid (insecure) key.